core/src/main/java/com/homeaway/streamplatform/streamregistry/configuration/TopicsConfig.java

```diff
@@ -25,7 +25,8 @@
 
     @Valid
     @NotNull
-    String producerTopic;
+    EventStoreTopic eventStoreTopic;
```

Comment:
Note: This is a backwards incompatible change

Reply:
yeah... worth calling out in configuration changes...

I assume it breaks if you don't "do the right thing"?

Reply:
yes it is a breaking change in ha-stream-registry , and i will make the corresponding PR in it as well.

Reply:
My point is that the version should be bumped accordingly.

Reply:
We are in 0.X which is understood to be not backward compatible.

If you are suggesting this should be a 0.5.x instead of 0.4.x... good callout.